### ISSUE TITLE
[PDKI-827] Data Mask AI - Coding Phase - EP - FInetune Pinpointing Adapter

### ISSUE THREAD

**Issue by jinijohn26** [5Plus-Test/jiramigrationtest#321]
> Migrated from Jira: [PDKI-827](https://pentadoc.atlassian.net/browse/PDKI-827)

---
### Checklist

- [x] export all annotated data from label studio
- [x] preprocess it into a JSON with VIony's script
- [x] update the finetuning script to be compatible with the data
- [x] find a way to show different images/entities to a model each epoch
- [x] finetune the adapter
- [x] save the code on github

---
### Comments

**Philipp Horn** (25/Apr/25 5:19 PM):
> The finetuning script is adjusted and now an adapter can be trained. The finished script is on GitHub.

It is set up to only use 20% of the data for now.
When using 20% of the data, it takes between 3 to 4 hours to train for 3 epochs.

Next different parameters for training can be tested.
